requirements/requirements.txt

```diff
@@ -34,7 +34,7 @@ ipython-genutils==0.2.0
 isodate==0.5.4
 jdcal==1.3
 Mezzanine==4.1.0
-MySQL-python==1.2.3
+#MySQL-python==1.2.3
```

Comment:
Do we need to Uncomment this Mysql-python?

Reply:
Yes sir we need to commented this Mysql-python.

Reply:
And what did you choose over mysql-python?

Reply:
Sir i choose mysqlclient==1.4.6

Reply:
That means that was to work on your local and for production you don't touch the requirement unless it's not required .